app/domain/etl/ga/user_feedback_processor.rb

```diff
@@ -45,7 +45,7 @@ def load_metrics_query(date_to_s)
       UPDATE facts_metrics
       SET useful_no = s.useful_no,
           useful_yes = s.useful_yes,
-          satisfaction= s.useful_yes / (s.useful_yes + s.useful_no::float)
+          satisfaction= s.useful_yes / NULLIF((s.useful_yes + s.useful_no::float),0)
```

Comment:
missing a few spaces -
```suggestion
          satisfaction = s.useful_yes / NULLIF((s.useful_yes + s.useful_no::float), 0)
```